deployment.json

```diff
@@ -20,5 +20,7 @@
   "TIMELOCK_ARTIFACT": "out/Timelock.sol/Timelock.json",
   "BUNDLE_STORE_ARTIFACT": "out/RedisConfidentialStore.sol/BundleConfidentialStore.json",
   "ADDR_OVERRIDES": {
+  },
+  "CONSTRUCTOR_ARGS": {
```

Comment:
should we consider merging these two together since they both have the same key?
The value could be like a predefined structure, such as 
key : {
  addr: ...,
  args: ....,
  <potential other stuff>
}

Reply:
Done